src/containers/CreateDoc/pdf.ts

```diff
@@ -0,0 +1,28 @@
+import PdfMake from 'pdfmake/build/pdfmake';
+
+import { TDocumentDefinitions } from 'types';
+import { TCreatedPdf } from 'types';
+
+interface PdfDefinitions {
+    [pdfDefinition: string]: TDocumentDefinitions
+}
+
+const pdfDefinitions: PdfDefinitions = {
```

Comment:
Not sure if I understand this, `pdfDefinitions` is the interface for the const `pdfDefinitions` (we need to adjust those names). 

Then the interface has a SINGLE optional key `pdfDefinition` which should have a `TDocumentDefinitions`, but here in the const, we have 2 keys.

How does it work ?